src/README.md

```diff
@@ -1,45 +0,0 @@
-> **WARNING:** Chimera is in early development stage, it's not production ready.
->
-> Feedback is highly appreciated.
-
-
-# Introduction
-
-Chimera is a [Kubernetes Dynamic Admission Controller](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/)
-that validates incoming requests using policies written in WebAssembly.
```

Comment:
I'm against deleting entirely the `README` file. At the same time I also agree we should not repeat stuff over there, because it just increases the amount of things we have to maintain.

Why don't we add a README with the following **succinct** information:
  * This is the repo with the source code of <link to the book website>
  * If you want to know more about chimera go to <link of chimera website>